app-utils/testdisk/autobuild/defines

```diff
@@ -1,6 +1,6 @@
 PKGNAME=testdisk
 PKGSEC=utils
-PKGDEP="libjpeg-turbo libreiserfs ntfs-3g openssl qt-4"
+PKGDEP="libjpeg-turbo libreiserfs ntfs-3g openssl qt-5"
 PKGDES="Checks and undeletes partitions, and utility for photo recovery"
 
 AUTOTOOLS_AFTER="--without-ewf --enable-sudo --enable-qt"
```

Comment:
The resulting package does not seem to come with a GUI.

Reply:
testdisk only supports qt4, so we drop this commit.